### DISCUSSION TITLE
Vercel doesn't compress svg file

### DISCUSSION THREAD

**Discussion by ellemedit** [vercel/community#2924] [Bug Report]
### Page to Investigate

https://dev.app.henesis.io/static/wallet-avatars/WalletAvatar01.svg

### Steps to Reproduce

<img width="1310" alt="Screenshot 2023-06-20 at 1 40 38 PM" src="https://github.com/vercel/community/assets/2682865/3e2c7419-f51c-4b20-86af-065c2f26d2f7">
<img width="1309" alt="Screenshot 2023-06-20 at 1 43 48 PM" src="https://github.com/vercel/community/assets/2682865/8ac9f4ce-a4a1-4476-b4d0-c3c77025460c">

It looks the response should have content-encoding headers and the body should be compressed but none of them applied. Perhaps, https://vercel.com/docs/concepts/edge-network/compression#mime-types-vercel-edge-network-compress are not applied to next.js public directory?

```
fetch("https://dev.app.henesis.io/static/wallet-avatars/WalletAvatar01.svg", {
  "headers": {
    "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7",
    "accept-language": "en-US,en;q=0.9",
    "cache-control": "max-age=0",
    "if-none-match": "\"31789aa3b219e2da58f430bcfc853b7d\"",
    "sec-ch-ua": "\"Not.A/Brand\";v=\"8\", \"Chromium\";v=\"114\", \"Google Chrome\";v=\"114\"",
    "sec-ch-ua-mobile": "?0",
    "sec-ch-ua-platform": "\"macOS\"",
    "sec-fetch-dest": "document",
    "sec-fetch-mode": "navigate",
    "sec-fetch-site": "same-origin",
    "sec-fetch-user": "?1",
    "upgrade-insecure-requests": "1"
  },
  "referrerPolicy": "strict-origin-when-cross-origin",
  "body": null,
  "method": "GET",
  "mode": "cors",
  "credentials": "include"
});
```

**Comment by smaeda-ks** [comment]
@Beingbook We set the minimum length of a response that will be compressed, which is `1000` as of today. The length is determined only from the `Content-Length` response header field. Your SVG image is just under 1000 so it's skipped

**Comment by ellemedit** [comment]
thank you for letting me know
